google/cloud/spanner/internal/connection_impl.cc

```diff
@@ -1139,6 +1139,11 @@ StatusOr<spanner::CommitResult> ConnectionImpl::CommitImpl(
   request.set_return_commit_stats(params.options.return_stats());
   request.mutable_request_options()->set_priority(
       ProtoRequestPriority(params.options.request_priority()));
+  if (params.options.max_commit_delay().has_value()) {
+    *request.mutable_max_commit_delay() =
+        google::protobuf::util::TimeUtil::MillisecondsToDuration(
+            params.options.max_commit_delay().value().count());
```

Comment:
We avoid `value()` when we know `has_value()`, so ...
```
params.options.max_commit_delay()->count()
```

Reply:
Done.